docs/wiki/Ethereum.md

```diff
@@ -0,0 +1,102 @@
+by the application.
+
+Unlike the other native wallets, the Ethereum wallet uses "RPC providers"
+specified by the user. The is the wallet's gateway to the Ethereum network.
+Casual ETH users may be familiar with Infura, as an example, since this is the
+default provider for the common MetaMask wallet. When creating your Ethereum
+wallet in the DEX application, you will need to specify the full path to an
+Ethereum RPC provider, i.e. an RPC endpoint.
+
+## Wallet Setup
+
+From the "Wallets" page of the DEX application, you select Ethereum from the
+left menu, which will show you your the ETH balance if you already have a
+wallet, or you will get a "Create a Ethereum Wallet" button:
+
+   <img src="images/eth-create-wallet.png" width="320">
+
+When you click the button, you will be able to specify one or more RPC provider
+endpoints. You can provide a single endpoint, but it is recommended to specify
+more than one for redundancy. Click the "+" icon at the right to add additional
+providers.
+
+   <img src="images/eth-wallet-settings.png" width="320">
+
+The types of endpoints that are supported are:
+
+- (**Recommended**) WebSocket (ws:// or wss:///). These use a persistent
+  connection, which is efficient and supports new block subscriptions to reduce
+  provider utilization and bandwidth.
+- HTTP (http:// or https://). These are less efficient and do not support new
+  block subscriptions, so the client frequently polls for new blocks.
+- IPC. These are typically a special file on your local machine. You must be
+  running a local node to use this, and the IPC file permissions must be set to
+  allow the DEX app to use it. However, typically even a full node will expose a
+  WebSocket interface.
+
+## RPC Provider Suggestions
+
+Consider the following tips when configuring your Ethereum wallet:
+
+- Use **multiple providers**, even if one of them is your own full node. This
+  provides redundancy. Often a provider will fall behind or even stall, so
+  multiple providers adds robustness to your wallet.
+
+- Get a custom RPC endpoint, with your own **private API key**. Many providers
+  will allow you to create a custom endpoint that includes an API "key" that is
+  assigned just to you.
+  
+  This is important since all providers have request rate limits, which may be
+  exhausted quickly with a generic RPC endpoint (e.g. https://rpc.ankr.com/eth)
+  that enforce rate limiting based on the source IP address. This is
+  particularly important if you use a VPN or other proxy where you share an
+  outbound IP address with many other people.
+
+  Further, taking the time to get an endpoint with your personal API key often
+  gives you access to a "dashboard" with the provider so you can monitor your
+  API requests.
+
+- Use **WebSockets** (wss:// URL scheme) when using a third-party RPC provider.
+  It is much more efficient than HTTP, the wallet will become aware of new
+  blocks sooner, and it will consume less of your quota with the provider.
```

Comment:
I believe the client will attempt to upgrade even if the user specifies http. Could be noted.